### PR TITLE
Styling tweaks for legibility

### PR DESCRIPTION
**Pull request by brianlevy**
An effort to improve legibility. Mostly css changes. In order to make the changes to the narrow view, added a span to the table cells which only gets a style added at the narrow view.

### CODE REVIEW

File: src/components/DataTable.css [hunk 1]
```diff
@@ -8,8 +8,13 @@
...
 	border: none; /* restyle at will */
 }
+:local(.narrow) td span {
+	display: inline-block;
+  	width: 4.5em;
+  	color: #999999;
+}
 :local(.narrow) td:nth-child(1) {
-	border-top: 1px dotted; /* this is the border between cells; restyle at will */
+	border-top: 2px solid #dddddd; /* this is the border between cells; restyle at will */
```
Comment (samuelcole) [inline src/components/DataTable.css]: you should remove the comment, since you did restyle it

File: src/components/DataTable.js [hunk 2]
```diff
@@ -83,7 +83,7 @@ export default class DataTable extends React.Component {
           <tbody>
             {values.map((val, i) => (
               <tr key={i}>
-                {columns.map((col, j) => <td key={j}>{wide ? '' : col + ':'} {val[col]}</td>)}
+                {columns.map((col, j) => <td key={j}><span>{wide ? '' : col + ':'}</span> {val[col]}</td>)}
```
Comment (samuelcole) [inline src/components/DataTable.js]: kinda weird that this is an empty span when the table is wide? i think

```
{wide ? '' : <span>{col}:</span>}
```

would work? maybe, i'm not sure about jsx inside ternaries

File: src/styles.css [hunk 3]
```diff
@@ -17,6 +17,12 @@ body {
   background: #fff;
 }
 
+.table-responsive tr td span {
+
```
Comment (samuelcole) [inline src/styles.css]: empty? can we get rid of the selector?

File: src/components/DataTable.js (hunk 2 above)
Comment (brianlevy) [inline src/components/DataTable.js]: Ideally the span would only be on the narrow view but I don't know how to do that. Any suggestions to achieve the same result are welcome.

File: src/styles.css (hunk 3 above)
Comment (brianlevy) [inline src/styles.css]: oops. missed that.

**Comment by samuelcole** [comment]
👍

**Comment by decause** [comment]
:+1: lgtm

**Pull request merged**